sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/text/TextPollOptionView.kt

```diff
@@ -210,19 +226,40 @@ internal class TextOptionView(context: Context?) : RelativeLayout(context) {
                 addRule(ALIGN_PARENT_LEFT)
             }
 
+            val maxVotingText = votePercentageText.apply {
+                text = "100%"
+            }
```

Comment:
aren't maxVotingText and votePercentageText going to be a reference to the same thing after? Use of the apply block here seems odd to me, because it looks like the text change is isolated to maxVotingText but the truth is the change is made to votePercentageText too.

Reply:
Yeah, they are. I agree it's weird, i'll change it